gpu4pyscf/hessian/uhf.py

```diff
@@ -292,7 +293,7 @@ def fvind_vo(mo1):
     avail_mem = get_avail_mem()
     # *8 for spin-up/down input dm, vj, vk, and vxc
     blksize = int(min(avail_mem*.3 / (8*3*nao*nao*8),
-                      avail_mem*.6 / (8*nmo*nocc*natm*3*5)))
+                      avail_mem*.3 / (8*nmo*nmo*3*6)))  # in vj, vk, dm in AO
```

Comment:
How is the memory estimation derived?

Reply:
The same reason as above, but for regular JK kernel. This still underestimates the size of temporary variables since they are in cartesian atomic basis. That's why only 0.3 avail_mem is used. But the two estimates should be merged.